src/lib.rs

```diff
@@ -21,8 +21,8 @@ enum Action {
     RemoveEntry(String),
 }
 
-/// `BTree` is where the informatio `Sender` is contained.
-/// Its inner implementation has a tuple containing the action to be taken as well as a oneshot channel to receive data.
+/// `BTree` is where the information `Sender` is contained.
+/// It's inner implementation has a tuple containing the action to be taken as well as a oneshot channel to receive data.
```

Comment:
Why did you change from its to it's? 

`its inner implementation` would mean something like `sua implementação interna`, while
`it's inner implementation` would mean something like `é a implementação interna`.

Here the correct would be `its`

Reply:
You're right, I missread it.